app/src/main/res/values/strings_settings.xml

```diff
@@ -30,81 +30,81 @@
     <string name="set_transparent_status_bar_hint">When favorites are closed</string>
     <string name="set_hide_menu_button">Hide the menu button</string>
-    <string name="set_hide_menu_button_help">It stays accessible using the launcher icon</string>
+    <string name="set_hide_menu_button_help">Always accessible using the launcher icon</string>
     <string name="set_icon_pack">Use an installed icon pack</string>
-    <string name="set_icon_pack_none">None (use the real icons)</string>
-    <string name="set_hide_app_names">Hide applications names</string>
-    <string name="set_hide_app_names_help">Folder names will still be displayed</string>
+    <string name="set_icon_pack_none">None (use default icons)</string>
+    <string name="set_hide_app_names">Hide app names</string>
+    <string name="set_hide_app_names_help">Folder names are always shown</string>
     <string name="set_remove_padding">Remove padding when names are hidden</string>
-    <string name="set_remove_padding_help">May produce unexpected results with long folders names</string>
+    <string name="set_remove_padding_help">Can cause problems with long folders names</string>
     <string name="set_force_portrait">Force portrait mode</string>
-    <string name="set_force_portrait_help">The launcher will not rotate even if automatic rotation is enabled</string>
+    <string name="set_force_portrait_help">Prevents the launcher rotating even if auto-rotation is on</string>
     <string name="set_immersive_mode">Use immersive mode</string>
     <string name="set_immersive_mode_help">Hide system bars (long press on home screen to force if needed)</string>
     <string name="set_reverse_interface">Reverse interface</string>
-    <string name="set_reverse_interface_help">Display the favorites panel on bottom</string>
-    <string name="error_appfilter_not_found">The appfilter.xml file was not found in the selected icon pack! (%1$s)</string>
+    <string name="set_reverse_interface_help">Show the favorites panel on bottom instead</string>
+    <string name="error_appfilter_not_found">Could not find appfilter.xml in the selected icon pack. (%1$s)</string>
 
     <!-- Export / Import -->
-    <string name="export_import_title">Export / Import</string>
-    <string name="export_import_help">These buttons allow you to export the application data (favorites, shortcuts and settings) or to import a previous export.</string>
+    <string name="export_import_title">Export and import</string>
+    <string name="export_import_help">These buttons allow exporting the app data (favorites, shortcuts and settings) or importing them again.</string>
     <string name="button_export">Export</string>
     <string name="button_import">Import</string>
-    <string name="export_import_header_internal_files">Internal text files content</string>
+    <string name="export_import_header_internal_files">Internal text file content</string>

```

Comment:
This one should stay plural I think.
It is used in the exported file as an header above the section about internal files (favorites.txt, hidden.txt, etc.).

---

**All the other suggestions for "strings_settings.xml" are ok for me and indeed really good improvements.
Thank you very much for this work!**

Reply:
"Files" becomes a verb then. "To file". The internal text files are filing the content.

Reply:
```suggestion
    <string name="export_import_header_internal_files">Content of internal text files</string>
```
The second one is indeed the most accurate, let's go for this!